clusters/hive/pools/hypershift/hypershift-ocp-4-8-0-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  name: hypershift-ocp-4-8-amd64-aws-us-east-1
+  namespace: hypershift-cluster-pool
+  labels:
+    product: ocp
+    version: "4.8"
+    architecture: amd64
+    cloud: aws
+    owner: hypershift
+    region: us-east-1
+spec:
+  baseDomain: hive.aws.ci.openshift.org
+  imageSetRef:
+    name: ocp-4.8.0-amd64
+  installConfigSecretTemplateRef:
+    name: install-config-aws-us-east-1
+  skipMachinePools: true
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: aws-creds-osd-hypershift
+      region: us-east-1
+  pullSecretRef:
+    name: pull-secret
```

Comment:
I will create this secret manually before https://issues.redhat.com/browse/DPTP-2353 is implemented.

Reply:
```
oc --context hive get secret -n hypershift-cluster-pool pull-secret
NAME          TYPE                             DATA   AGE
pull-secret   kubernetes.io/dockerconfigjson   1      79s
```

Done.